src/utils/validateEmail.ts

```diff
@@ -0,0 +1,4 @@
+export function validateEmail(email: string): boolean {
```

Comment:
change this function to an arrow function
use `export default` when there's only 1 thing to export.
use inline return `() => return_value` and remove the variable and use it in place